SMPyBandits/Arms/Binomial.py

```diff
@@ -59,6 +59,12 @@ def draw_nparray(self, shape=(1,)):
         """ Draw a numpy array of random samples, of a certain shape."""
         return np.asarray(npbinomial(self.draws, self.probability, shape), dtype=float)
 
+    def set_mean_param(self, probability, draws=None):
+        if draws==None:
+            self.draws = draws
+        self.mean = probability
+        self.probability = probability * self.draws
```

Comment:
I don't understand this code, if `self.draws` is set to `None`, then `self.probability` can be `None` ?

Reply:
this is a bug --> it is 
```
if draws is not None:
```
thanks !